src/server/on-off-farm/index.njk

```diff
@@ -0,0 +1,56 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends 'layouts/page.njk' %}
+
+{% block metaBeforeContent %}
+  {{ govukBackLink({
+    text: "Back",
+    href: "/"
```

Comment:
I think this one needs a bit of a review with Guy - I can see that the component out-of-the-box specifies a href rather than tapping into javascript history API. 

The guidance & implementation are a bit confusing: 

> Make sure the link takes users to the previous page they were on, in the state they last saw it.

does not match the offered functionality, looking at https://design-system.service.gov.uk/components/back-link/.

Unless we pick up the HTTP referrer & populate it in here?  Which would work with javascript & match the spirit of the guidance (but not the examples)